common.tf

```diff
@@ -89,9 +89,15 @@ module "dr2_kms_key" {
       module.e2e_tests_ecs_task_role.role_arn,
       module.copy_tna_to_preservica_role.role_arn,
       "arn:aws:iam::${module.tre_config.account_numbers["prod"]}:role/prod-tre-editorial-judgment-out-copier",
```

Comment:
1. Could this be saved to a variable?
2. Is it fine that we are using the hardcoded "prod" in both places?